hack/README.md

```diff
@@ -0,0 +1,31 @@
+# Integration testing against different Kubernetes versions
+
+**TODO: edit repo links based on final location of this README**
+
+## Prerequisites
+
+In order to run the following, you need:

```

Comment:
"in order to run make e2etest"

Reply:
Check new wording/layout in CONTRIBUTING.md, but should be better?